core/services/synchronization/presenters.go

```diff
@@ -138,11 +139,11 @@ type syncInitiatorPresenter struct {
 }
 
 type syncTaskRunPresenter struct {
-	Index                int         `json:"index"`
-	Type                 string      `json:"type"`
-	Status               string      `json:"status"`
-	Error                null.String `json:"error"`
-	Result               interface{} `json:"result,omitempty"`
-	Confirmations        uint64      `json:"confirmations"`
-	MinimumConfirmations uint64      `json:"minimumConfirmations"`
+	Index                int           `json:"index"`
+	Type                 string        `json:"type"`
+	Status               string        `json:"status"`
+	Error                null.String   `json:"error"`
+	Result               interface{}   `json:"result,omitempty"`
+	Confirmations        clnull.Uint32 `json:"confirmations"`
```

Comment:
`omitempty`?

Reply:
I left it because I felt having a key with a `null` value does convey that it has that field but it's been unset. Hiding the field altogether makes discoverability harder, and makes it more awkward when all of a sudden no key because `confirmations: 1` the moment it decides to get set, since it is a volatile value.

Very indifferent about this, so just let me know if you still want `omitempty`.